bridges/snowbridge/pallets/ethereum-client/src/tests.rs

```diff
@@ -595,7 +547,7 @@ fn submit_update_with_missing_bootstrap() {
 	new_tester().execute_with(|| {
 		assert_err!(
 			EthereumBeaconClient::submit(RuntimeOrigin::signed(1), update),
-			Error::<Test>::NotBootstrapped
+			Error::<Test>::SkippedSyncCommitteePeriod
```

Comment:
Seems like a side-effect of always importing the inbound test fixtures - now we can't test the `NotBootstrapped` error.